pages/linux/objcopy.md

```diff
@@ -0,0 +1,24 @@
+# objcopy
+
+> The GNU objcopy utility copies the contents of an object file to another.
+> More information: <https://manned.org/objcopy>.
+
+- Copy data to another file:
+
+`objcopy {{/your/path/to/file/src}} {{/your/path/to/file/destination}}`
```

Comment:
```suggestion
`objcopy {{path/to/source_file}} {{path/to/target_file}}`
```
The [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides%2Fstyle-guide.md) recommends:
- Starting placeholders for **absolute** path arguments with "/" and _**not** starting with "/" for **relative** path arguments_
- File placeholders in the form "{{path/to/file}}", where file can be relaced with a more descriptive name